posgram/Service/IService/IUserService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using posgram.Models;
+
+namespace posgram.Service.IService
+{
+    public interface IUserService
+    {
+        Task<List<User>> GetAll();
+        Task<User> GetById(long id);
+        Task Create(User entity);
+        Task Update(User entity);
+        Task Delete(long id);
+        Task<List<Post>> GetPostsById(long id);
```

Comment:
I think, `ById` is redundant:
```suggestion
        Task<List<Post>> GetPosts(long id);
```

Reply:
We have both methods. One provides posts by one user, and the other provides a feed of posts. 
What do you think about this?

Reply:
Ok, we need both methods